setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = django-sass-processor
+version = attr: sass_processor.__version__
+description = SASS processor to compile SCSS files into *.css, while rendering, or offline.
+long_description = file: README.md
+long_description_content_type = markdown
+author = Jacob Rief
+author_email = jacob.rief@gmail.com
+url = https://github.com/jrief/django-sass-processor
+license = MIT
+keywords = django, sass
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: Web Environment
+    Framework :: Django
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Topic :: Internet :: WWW/HTTP :: Dynamic Content
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Framework :: Django :: 2.2
+    Framework :: Django :: 3.0
+    Framework :: Django :: 3.1
+
+[options]
+include_package_data = True
+packages = sass_processor
```

Comment:
setuptools does not include sub-packages automatically, so you’re going to miss things here.

You can translate `find_packages(exclude=['tests'])` from setup.py to setup.cfg, see setuptools docs.

Reply:
There are no sub-packages here, and `sass_processor` is the only package. I have verified both the sdist and the resulting installation, and both are complete.

Can you point me to anything in particular that you find missing?

Reply:
I see these sub-packages in the repo:
sass_processor.jinja2
sass_processor.management
sass_processor.management.coomands
sass_processor.templatetags

They are included in the distribution because of MANIFEST.in, but it’s not the proper way.

Reply:
Ah, that comes from there.

I must admit I am still a bit confused by the concept of "packages" at some times (do they require an `__init__.py` to be considered packages now, or not?).

As the released version still has valid distribution files (even if they were generated in a non-standard way), I do not consider this iportant for this release, however.

Reply:
Yes, they need `__init__.py`.  Namespace packages don’t, but these are for a different use case.